README.md

```diff
@@ -39,9 +43,10 @@ The following ENV vars must be set in your local env for this script to run corr
 
 |ENV VAR|explanataion|Nominal value|
 |---|---|---|
-|INSTANCE_NAME|Name of the instance you wish to communicate with|datascience_base|
-|REMOTE_DATALABS_PATH|Path to the the datalabs folder on the remote instance|/data/datalabs|
-|LOCAL_DATALABS_PATH|Path to the datalabs folder on your local machine|/home/matthew/Documents/wellcome/datalabs|
+|INSTANCE_NAME|Name of the instance you wish to communicate with. Note that this requires that the tag `Name` is set on the instance of interest|datascience_base|
+|PREFIX|Prefix used to identify instances for a given task|datascience|
```

Comment:
possilbly `filter_instances` or `filter_prefix` a more descriptive name

Reply:
Sure, that's better

Reply:
Resolved in https://github.com/wellcometrust/remote/pull/8/commits/4b056c1733549f80b7ebbd46467638fb98701ee7